lib/project_types/script/messages/messages.rb

```diff
@@ -126,10 +126,10 @@ module Messages
         },
 
         project_deps: {
-          deps_are_installed: "{{v}} Dependencies installed",
-          installing_with_npm: "Installing dependencies with npm",
+          deps_are_installed: "{{v}} None required",
```

Comment:
Since we're changing the messages here, perhaps we should be also updating the keys to keep them in sync?